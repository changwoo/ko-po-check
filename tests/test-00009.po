# -*-text-*-
# test 9 - KDE
#
msgid ""
msgstr ""
"Project-Id-Version: test 9\n"
"POT-Creation-Date: 2002-08-01 01:30+0900\n"
"PO-Revision-Date: 2002-08-01 01:30+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

# E: 번역자 써야 함
#: test.c:9
msgid ""
"_: NAME OF TRANSLATORS\n"
"Your names"
msgstr "_: 옮긴이의 이름"
"류창우"

# E: 번역자 써야 함
#: test.c:9
msgid ""
"_: NAME OF TRANSLATORS\n"
"Your names"
msgstr "옮긴이의 이름"
"류창우"

# E: 번역자 메일 써야 함
#: test.c:9
msgid ""
"_: EMAIL OF TRANSLATORS\n"
"Your email"
msgstr "_: 번역한 사람 이메일"
"cwryu@debian.org"

# E: 번역자 메일 써야 함
#: test.c:9
msgid ""
"_: EMAIL OF TRANSLATORS\n"
"Your email"
msgstr "옮긴이의 이메일"
"cwryu@debian.org"

# E: 번역자 메일 써야 함
#: test.c:9
msgid ""
"_: Low Contrast\n"
"Low"
msgstr ""
"_: 저채도\n"
"낮음"
