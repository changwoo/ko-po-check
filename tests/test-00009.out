test-00009.po:20: KDE에서 "_:"부터 첫 번째 \n까지는 본디말에 대한 설명이므로 번역하면 안 됩니다.
test-00009.po:28: 번역자 이름을 써야 합니다. "_:"로 시작하는 문장은 번역하면 안 됩니다.
test-00009.po:36: KDE에서 "_:"부터 첫 번째 \n까지는 본디말에 대한 설명이므로 번역하면 안 됩니다.
test-00009.po:44: 번역자 이메일을 써야 합니다. "_:"로 시작하는 문장은 번역하면 안 됩니다.
test-00009.po:52: KDE에서 "_:"부터 첫 번째 \n까지는 본디말에 대한 설명이므로 번역하면 안 됩니다.
